docs/concepts/flox-vs-containers.md

```diff
@@ -0,0 +1,452 @@
+---
+Title: Flox vs. container workflows
+Description: How does a Flox workflow differ from a container workflow?
+---
+
+# Flox vs. container workflows
+
+Containers are everywhere these days.
+They're the de-facto deployment method in the industry,
+and they're often used for local development as well to ensure that every
+developer gets the same environment.
+
+Flox gives you these same benefits without many of the papercuts,
+but the workflow is slightly different so it's worth exploring.
+
+## Create a new development environment
+
+Let's say you've created a new directory for your project,
+`myproject`,
+and entered it:
+
+```bash
+$ mkdir myproject
+$ cd myproject
+```
+
+=== "Flox"
+
+    Create a Flox environment for the project via [`flox init`][init]:
+
+    ```bash
+    $ flox init
+    ```
+
+    This creates a `.flox` directory in `myproject`.
+    At this point you can enter the environment,
+    but it doesn't provide any new packages or functionality.
+
+=== "Containers"
+
+    Create a new `Dockerfile`.
+
+    ```bash
+    $ touch Dockerfile
+    ```
+
+    At this point the `Dockerfile` isn't really useable since it doesn't say which
+    image to build on top of (there's no `FROM` line).
+    Let's pick the latest Ubuntu LTS release.
+
+    ```bash
+    $ echo "FROM ubuntu:noble" >> Dockerfile
+    ```
+
+    This already adds some variability in that which image this refers to may
+    change from moment to moment.
+
+---
+
+## Add packages
+
+Now let's install some packages.
+In a typical project there's usually different subsets of packages:
```

Comment:
**nitpick:** There's a subject–verb agreement issue here, though the tone is conversational so it may not matter.
```suggestion
In a typical project there are usually different subsets of packages:
```